aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,63 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
 
-.. note::
+ The follwoing instrcution were tested using CentOS 7.
+ 
+Install Mono
```

Comment:
But on CentOS minimal, the yum-utiels are not installed. In this point you run in a error if you try install mono with the official doku. I think it's good to include the link to the mono doku but I think it's better we have a short install statement.